omniNotes/src/androidTest/java/it/feio/android/omninotes/tests/HamburgerTest.java

```diff
@@ -74,5 +74,20 @@ public void testHamburgerValidation() throws InterruptedException {
 
         // Verify: List View has one child
         assertTrue("List View has more then one child", Trash.isListViewWithOneChild());
```

Comment:
I think that you can use assertEquals here, instead of that function , you can use getListViewChildCount directly.
Eg. assertEquals( "message", expected result :1, actual: getListViewChildCount(matcher)); .

Reply:
Done.